s3-sqs-lambda-terraform/README.md

```diff
@@ -0,0 +1,81 @@
+# Amazon S3 to Amaozn SQS queue to AWS Lambda
```

Comment:
```suggestion
# Amazon S3 to Amazon SQS queue to AWS Lambda
```

Reply:
Typo fixed, thanks.

example-pattern.json file added as an attachment on initial post on these PR. Do you want me do add it on the commit?